.github/workflows/build.yml

```diff
@@ -121,6 +138,25 @@ jobs:
         # There no cmake install presets so install in traditional way
         run: cmake --install builds/${{ matrix.os.preset }}/ --config ${{ matrix.build_type }}
 
+      - name: Sign, package, and notarize for macOS
+        if: ${{ matrix.os.preset == 'mac' && env.MACOS_SIGNING == 'true' }}
+        uses: halprin/macos-sign-package-notarize@v2
```

Comment:
same as above, passing secrets into an action not hosted exclusively by @DescentDevelopers/d3-maintainers theoretically poses a risk. long-term we should probably fork this and take control of it.

Reply:
I forked both under the DescentDevs namespace, @halprin  should have access to both when they accept the organization invite.

https://github.com/DescentDevelopers/import-codesign-certs
https://github.com/DescentDevelopers/macos-sign-package-notarize

Is there anything more we need to do to make the repositories a github action?

Reply:
@tophyr : "sOmEoNe ShOuLd Do ThIs"
also @tophyr : _doesn't do it himself_
@Lgt2x : "...ok"

@halprin i'm gonna request changes on this PR to use the @DescentDevelopers versions of those actions, but the actions themselves look fine

Reply:
It worked, surprisingly.  Also, that `halprin` guy is pretty shady who made that https://github.com/halprin/macos-sign-package-notarize action.  Anyway, fixed!